### PR TITLE
docs(elevation): add guide for elevation helpers

### PR DESCRIPTION
**Pull request by willshowell**
Credit to @M-a-c for initial version (https://github.com/angular/material2/pull/3917)

Fixes https://github.com/angular/material2/issues/2800

cc @Splaktar @mmalerba

### CODE REVIEW

File: guides/elevation.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+```scss
+@import '~@angular/material/theming';
+
+.my-class {
+  @include mat-elevation(2);
+}
+```
+
+For convenience, you can use the `mat-elevation-transition` mixin to add a transition between
+elevation changes:
```
Comment (mmalerba) [inline guides/elevation.md]: ... add a transition when the elevation changes

File: guides/elevation.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+.my-class {
+  @include mat-elevation-transition;
+  @include mat-elevation(2);
+
+  &:active {
+    @include mat-elevation(8);
+  }
+}
+```
```
Comment (mmalerba) [inline guides/elevation.md]: should we add an embedded example here too?

Comment (willshowell) [reply guides/elevation.md]: It won't export very well to plunker will it?

Comment (mmalerba) [reply guides/elevation.md]: Oh right, I forgot our examples use plain CSS

**Pull request merged**